gulp/tasks/favicon.js

```diff
@@ -0,0 +1,65 @@
+module.exports = function (gulp, plugins, config) {
+  return function javascript(cb) {
+    plugins.realFavicon.generateFavicon({
+      masterPicture: 'assets/logo.svg',
+      dest: 'assets/favicon',
```

Comment:
Hi should `assets/favicon` not be in the `.gitignore` and the files with these path not in this repo?